### PR TITLE
Update MIGRATION.md

### PR DESCRIPTION
**Pull request by YKalashnikov**
The link for webpack migration was incorrect

Closes #

## Checklist

- [ ] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [ ] Make sure to add/update documentation regarding your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

- [ ] If this PR should be tested against many or all sandboxes,
      make sure to add the `ci:merged` or `ci:daily` GH label to it.
- [ ] Make sure this PR contains **one** of the labels below.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -4181,7 +4181,7 @@ We're sorry if this inconveniences you, we have tried via this document and prov
 
 Storybook will now use webpack 2 (and only webpack 2).
```
Comment (kbrilla) [inline MIGRATION.md:4182]: webpack 2 or 5?

Comment (jonniebigodes) [reply MIGRATION.md:4182]: @kbrilla, thanks for the comment and your question. Appreciate it 🙏 . Actually, it's Webpack 2, this pull request and change is referring to Storybook version 3.x which was released a long time ago.

Comment (kbrilla) [reply MIGRATION.md:4182]: `migrate/5` took me by surprise here - thank You for clarification

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@YKalashnikov I left one item for you to follow up on and would like to hear your thoughts on it when you're able.

Let me know and we'll go from there.

I hope you have a great day.

Stay safe

File: MIGRATION.md [hunk 2]
```diff
@@ -4181,7 +4181,7 @@ We're sorry if this inconveniences you, we have tried via this document and prov
...
 If you are using a custom `webpack.config.js` you need to change this to be compatible.
-You can find the guide to upgrading your webpack config [on webpack.js.org](https://webpack.js.org/guides/migrating/).
+You can find the guide to upgrading your webpack config on [webpack.js.org](https://webpack.js.org/migrate/5/).
```
[the remark this answers is not in the document]
Comment (YKalashnikov) [reply MIGRATION.md:4184]: Hello @jonniebigodes yes for some reason i was thinking the correct webpack version is 5

[the remark this answers is not in the document]
Comment (jonniebigodes) [reply MIGRATION.md:4184]: @YKalashnikov thanks for following up with me and providing context on this. If you're ok with it we should close this pull request and go from there. Sounds good?

[the remark this answers is not in the document]
Comment (YKalashnikov) [reply MIGRATION.md:4184]: Thanks @jonniebigodes

[the remark this answers is not in the document]
Comment (jonniebigodes) [reply MIGRATION.md:4184]: No need to thank whatsoever @YKalashnikov. It was my pleasure to review and provide some guidance on this pull request.

**Pull request closed without merging**